wiki/Ranking_Criteria/osu!catch/en.md

```diff
@@ -1,9 +1,11 @@
 # osu!catch ranking criteria
 
-***Notice: This article is an extension of the [general ranking criteria](/wiki/Ranking_Criteria).***
+***Note: This article is an extension of the [general ranking criteria](/wiki/Ranking_Criteria).***
```

Comment:
according to the ASC, this should be `Notice` actually

```
- `Note` should be used for factual or trivial details.
- `Notice` should be used for reminders or to draw attention to something that the reader should be made aware of.
```

Reply:
I often mix these two up to a point it sometimes becomes inconvenient